src/data/main-nav.json

```diff
@@ -92,5 +92,24 @@
       "url": "../assets/img/hero2-large.jpeg",
       "description": "image-hero"
     }
+  },
+  "header2": {
+    "header": "SmartVoice",
+    "description": "Enjoy peace of mind and remotely monitor, record, and access HD video of your business.",
+    "cta": {
+      "label": "Get this plan",
+      "url": "/smartwifi"
+    },
+    "images": {
+      "small": {
+        "url": "../assets/img/hero-small.jpg"
+      },
+      "medium": {
+        "url": "../assets/img/hero-medium.jpg"
+      },
+      "large": {
+        "url": "../assets/img/hero-large.jpg"
+      }
+    }
   }
 }
```

Comment:
Remember that you shouldn't have a file that is holding data just for an specific component, but instead, a file that contains data for a whole page.